tests/queries/0_stateless/02891_rename_table_without_keyword.reference

```diff
@@ -0,0 +1,7 @@
+r1
+r2
+r3
+test_dictionary
+test_dictionary_2
+rename_db
+rename_db_2
```

Comment:
```suggestion
test_dictionary
rename_db
rename_db
```

Reply:
I've tried rename dictionary with statement `reanem table dictionary_name to dictionary_name_2` and I found this sql can be exeuted without exception. So It will not throw an exception when you rename dictionary use sql `rename dictionary_name to dictionary_name_2` in this PR. Is that ok?